lib/config/executor/container_vm.go

```diff
@@ -81,6 +81,18 @@ type Diagnostics struct {
 	// ExitLogs is a best effort record of the time of process death and the cause for
 	// restartable entities
 	ExitLogs []ExitLog `vic:"0.1" scope:"read-write" key:"exitlogs"`
+
+	// SyslogConfig holds configuration for connecting to a syslog
+	// server
+	SysLogConfig *SysLogConfig `vic:"0.1" scope:"read-only" key:"syslog"`
+}
+
+// SyslogConfig holds the configuration necessary to connect to a syslog server
+type SysLogConfig struct {
```

Comment:
There is certainly a way to configure logrus hooks so that they are only called for certain log levels. How we expose that to the user is the difficult part here.